site/snippets/react/app.tsx

```diff
@@ -0,0 +1,15 @@
+import { AlchemyAccountProvider } from "@alchemy/aa-alchemy/react";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { config } from "./config";
+
+const queryClient = new QueryClient();
+
+export function App() {
+  return (
+    <QueryClientProvider client={queryClient}>
+      <AlchemyAccountProvider config={config} queryClient={queryClient}>
```

Comment:
does this imply that react query is a required dependency for our provider? are there other client impl types we would want to support to make the provider more generic?

Reply:
yea react query is a requirement and it's listed as a peerdep